CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+cmake_minimum_required(VERSION 2.8)
+
+cmake_policy(SET CMP0048 NEW)
+
+project(safeside VERSION 0.1.0 LANGUAGES CXX)
+
+add_library(safeside demos/cache_sidechannel.cc demos/instr.cc)
+
+# Spectre V1 -- bounds check bypass
+add_executable(spectre_v1 demos/spectre_v1.cc)
+target_link_libraries(spectre_v1 safeside)
+
+# Spectre V3 / Meltdown
```

Comment:
The Meltdown demo (and its associated Linux kernel module) are only expected to be useful on Linux. Let's hide both of those behind `if (${CMAKE_SYSTEM_NAME} STREQUAL "Linux")` so this build file can work on other platforms.

Reply:
Updated.